### PR TITLE
refactor: improve types and add comments

### PR DESCRIPTION
**Pull request by armano2**
issue https://github.com/babel/babel/issues/6679 got solved in babel and after next release we should enable alignment tests

### CODE REVIEW

File: src/node-utils.ts [hunk 1]
```diff
@@ -322,14 +327,11 @@ function canContainDirective(node: ts.Node): boolean {
 /**
  * Returns line and column data for the given ts.Node or ts.Token,
  * for the given AST
- * @param  {ts.Token|TSNode} nodeOrToken the ts.Node or ts.Token
```
Comment (armano2) [inline src/node-utils.ts]: in typescript there is no difference between node and token, both of types implements `ts.Node`

Comment (JamesHenry) [reply src/node-utils.ts]: Yeah this signature is a little odd, I agree. Would you mind tidying up the whole thing?

Comment (armano2) [reply src/node-utils.ts]: ok i will do so :)

**Pull request merged**